agave_app/aboutDialog.cpp

```diff
@@ -0,0 +1,94 @@
+#include "aboutDialog.h"
+
+#include <QApplication>
+#include <QDialogButtonBox>
+#include <QLabel>
+#include <QTextBrowser>
+#include <QVBoxLayout>
+
+AboutDialog::AboutDialog()
+  : QDialog()
+{
+  setWindowTitle("About AGAVE");
+  auto layout = new QVBoxLayout(this);
+  auto label = new QLabel(this);
+
+  label->setText("<h1><b>AGAVE</b></h1>"
+                 "<i>Advanced GPU Accelerated Volume Explorer</i>"
+                 "<h4><b>Version " +
+                 qApp->applicationVersion() +
+                 "</b></h4>"
+                 "");
+  label->setAlignment(Qt::AlignCenter);
+  layout->addWidget(label);
+
+  label = new QLabel(this);
+  label->setText(
+    "<a href=\"https://forum.image.sc/tag/agave\">Support forum</a><br/>"
+    "<a href=\"https://allen-cell-animated.github.io/agave\">Online documentation</a><br/>"
+    "<a href=\"https://github.com/allen-cell-animated/agave/issues\">Report a bug or make a request</a><br/>"
+    "<a href=\"https://github.com/allen-cell-animated/agave\">Source code</a><br/>"
+    "");
+  label->setTextInteractionFlags(Qt::TextBrowserInteraction);
+  label->setOpenExternalLinks(true);
+  layout->addWidget(label);
+
+  QString agaveUrl = "https://github.com/allen-cell-animated/agave";
+  auto text = new QTextBrowser(this);
+  text->setText(
+    "AGAVE is a desktop volume viewer that uses path trace rendering and cinematic lighting techniques to generate "
+    "images with a high degree of resolution and clarity.<br>"
+    "<a href=\"https://www.allencell.org/pathtrace-rendering.html\">Visit our website</a> to learn more and download "
+    "the latest version.<br>"
+    "<br>"
+    "If you use AGAVE in your research, please cite the following:<br>"
+    "<br/>"
+    "Daniel Toloudis, AGAVE Contributors (2023). AGAVE: Advanced GPU Accelerated Volume Explorer (Version " +
+    qApp->applicationVersion() +
+    ")"
+    "[Computer software]. Allen Institute for Cell Science. <a "
+    "href=\"" +
+    agaveUrl +
+    "\"></a><br/>"
+    "<br/>"
```

Comment:
Nit: I think it might be a little easier to use the citations if we put the citation information (both this and the bibtex info) in their own `QTextBrowser` block, and hoist the rest of the about text out to be on the same level as the links? That way, users can double click to copy the full citation information within the blocks.

Reply:
I redid all this after seeing Travis's idea to put citation info in a separate dialog.  Citation is now a help menu item.